Blank line at start of file causes read to fail

```
================================== FAILURES ===================================
__________________________ test_blank_line_at_start ___________________________

    def test_blank_line_at_start():
>       las = read(egfn('blank_line_start.las'))

tests\test_read.py:186:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
lasio\__init__.py:43: in read
    return LASFile(file_ref, **kwargs)
lasio\las.py:78: in __init__
    self.read(file_ref, **read_kwargs)
lasio\las.py:130: in read
    ignore_header_errors=ignore_header_errors)
lasio\las.py:117: in add_section
    raw_section = self.match_raw_section(pattern)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <lasio.las.LASFile object at 0x000000000743AF28>, pattern = '~V'
re_func = 'match', flags = <RegexFlag.IGNORECASE: 2>

    def match_raw_section(self, pattern, re_func="match", flags=re.IGNORECASE):
        '''Find raw section with a regular expression.

            Arguments:
                pattern (str): regular expression (you need to include the tilde)

            Keyword Arguments:
                re_func (str): either "match" or "search", see python ``re`` module.
                flags (int): flags for :func:`re.compile`

            Returns:
                dict

            Intended for internal use only.

            '''
        for title in self.raw_sections.keys():
>           title = title.strip()
E           AttributeError: 'NoneType' object has no attribute 'strip'

lasio\las.py:349: AttributeError
-------------------------------- Captured log ---------------------------------
reader.py                  202 DEBUG    get_encoding Using cchardet
reader.py                  214 DEBUG    cchardet method detected encoding of ASCII at confidence 1.0
reader.py                  152 INFO     Opening C:\Users\kent\Code\lasio\tests\examples\blank_line_start.las as ASCII and treating errors with "replace"
reader.py                  388 DEBUG    using read policy of "default"
reader.py                  391 DEBUG    adding substitution comma-decimal-mark
reader.py                  391 DEBUG    adding substitution run-on(-)
reader.py                  391 DEBUG    adding substitution run-on(.)
reader.py                  391 DEBUG    adding substitution run-on(NaN.)
reader.py                  414 DEBUG    added regexp substitution: pattern=re.compile('(\\d),(\\d)') substr="\1.\2"
reader.py                  414 DEBUG    added regexp substitution: pattern=re.compile('(\\d)-(\\d)') substr="\1 -\2"
reader.py                  414 DEBUG    added regexp substitution: pattern=re.compile('-?\\d*\\.\\d*\\.\\d*') substr=" NaN NaN "
reader.py                  414 DEBUG    added regexp substitution: pattern=re.compile('NaN[\\.-]\\d+') substr=" NaN NaN "
reader.py                  388 DEBUG    using null policy of "common"
reader.py                  391 DEBUG    adding substitution NULL
reader.py                  395 DEBUG    located substition for LAS.version.NULL as True
reader.py                  391 DEBUG    adding substitution (null)
reader.py                  391 DEBUG    adding substitution -
reader.py                  391 DEBUG    adding substitution 9999.25
reader.py                  391 DEBUG    adding substitution 999.25
reader.py                  391 DEBUG    adding substitution NA
reader.py                  391 DEBUG    adding substitution INF
reader.py                  391 DEBUG    adding substitution IO
reader.py                  391 DEBUG    adding substitution IND
reader.py                  411 DEBUG    added numerical substitution: None
reader.py                  414 DEBUG    added regexp substitution: pattern=re.compile(' \\(null\\)') substr=" NaN"
reader.py                  414 DEBUG    added regexp substitution: pattern=re.compile('\\(null\\) ') substr="NaN "
reader.py                  414 DEBUG    added regexp substitution: pattern=re.compile(' \\(NULL\\)') substr=" NaN"
reader.py                  414 DEBUG    added regexp substitution: pattern=re.compile('\\(NULL\\) ') substr="NaN "
reader.py                  414 DEBUG    added regexp substitution: pattern=re.compile(' null') substr=" NaN"
reader.py                  414 DEBUG    added regexp substitution: pattern=re.compile('null ') substr="NaN "
reader.py                  414 DEBUG    added regexp substitution: pattern=re.compile(' NULL') substr=" NaN"
reader.py                  414 DEBUG    added regexp substitution: pattern=re.compile('NULL ') substr="NaN "
reader.py                  414 DEBUG    added regexp substitution: pattern=re.compile(' -+ ') substr=" NaN "
reader.py                  411 DEBUG    added numerical substitution: -9999.25
reader.py                  411 DEBUG    added numerical substitution: 9999.25
reader.py                  411 DEBUG    added numerical substitution: -999.25
reader.py                  411 DEBUG    added numerical substitution: 999.25
reader.py                  414 DEBUG    added regexp substitution: pattern=re.compile('(#N/A)[ ]') substr="NaN "
reader.py                  414 DEBUG    added regexp substitution: pattern=re.compile('[ ](#N/A)') substr=" NaN"
reader.py                  414 DEBUG    added regexp substitution: pattern=re.compile('(-?1\\.#INF)[ ]') substr="NaN "
reader.py                  414 DEBUG    added regexp substitution: pattern=re.compile('[ ](-?1\\.#INF[0-9]*)') substr=" NaN"
reader.py                  414 DEBUG    added regexp substitution: pattern=re.compile('(-?1\\.#IO)[ ]') substr="NaN "
reader.py                  414 DEBUG    added regexp substitution: pattern=re.compile('[ ](-?1\\.#IO)') substr=" NaN"
reader.py                  414 DEBUG    added regexp substitution: pattern=re.compile('(-?1\\.#IND)[ ]') substr="NaN "
reader.py                  414 DEBUG    added regexp substitution: pattern=re.compile('[ ](-?1\\.#IND[0-9]*)') substr=" NaN"
reader.py                  284 DEBUG    Data section ["array"].shape = (24,)
============================== warnings summary ===============================
:171
  'pytest_runtest_makereport' hook uses deprecated __multicall__ argument

None
  pytest_funcarg__caplog: declaring fixtures using "pytest_funcarg__" prefix is deprecated and scheduled to be removed in pytest 4.0.  Please remove the prefix and use the @pytest.fixture decorator instead.
  pytest_funcarg__capturelog: declaring fixtures using "pytest_funcarg__" prefix is deprecated and scheduled to be removed in pytest 4.0.  Please remove the prefix and use the @pytest.fixture decorator instead.

-- Docs: http://doc.pytest.org/en/latest/warnings.html
============== 1 failed, 121 passed, 3 warnings in 10.84 seconds ==============
```

--- a/lasio/reader.py
+++ b/lasio/reader.py
@@ -257,6 +257,8 @@ def read_file_contents(file_obj, regexp_subs, value_null_subs,
 
     for i, line in enumerate(file_obj):
         line = line.strip()
+        if not line:
+            continue
         if line.startswith('~A'):
             # HARD CODED FOR VERSION 1.2 and 2.0; needs review for 3.0
             # We have finished looking at the metadata and need

--- a/tests/examples/blank_line_start.las
+++ b/tests/examples/blank_line_start.las
@@ -1,0 +1,47 @@
+
+~VERSION INFORMATION
+ VERS.                  1.2:   CWLS LOG ASCII STANDARD -VERSION 1.2
+ WRAP.                  NO:   ONE LINE PER DEPTH STEP
+~WELL INFORMATION BLOCK
+#MNEM.UNIT       DATA TYPE    INFORMATION
+#---------    -------------   ------------------------------
+ STRT.M        1670.000000:
+ STOP.M        1660.000000:
+ STEP.M            -0.1250:
+ NULL.           -999.2500:
+ COMP.             COMPANY:   # ANY OIL COMPANY LTD.
+ WELL.                WELL:   ANY ET AL OIL WELL #12
+ FLD .               FIELD:   EDAM
+ LOC .            LOCATION:   A9-16-49-20W3M
+ PROV.            PROVINCE:   SASKATCHEWAN
+ SRVC.     SERVICE COMPANY:   ANY LOGGING COMPANY LTD.
+ DATE.            LOG DATE:   25-DEC-1988
+ UWI .      UNIQUE WELL ID:   100091604920W300
+~CURVE INFORMATION
+#MNEM.UNIT      API CODE      CURVE DESCRIPTION
+#---------    -------------   ------------------------------
+ DEPT.M                      :  1  DEPTH
+ DT  .US/M     		     :  2  SONIC TRANSIT TIME
+ RHOB.K/M3                   :  3  BULK DENSITY
+ NPHI.V/V                    :  4   NEUTRON POROSITY
+ SFLU.OHMM                   :  5  RXO RESISTIVITY
+ SFLA.OHMM                   :  6  SHALLOW RESISTIVITY
+ ILM .OHMM                   :  7  MEDIUM RESISTIVITY
+ ILD .OHMM                   :  8  DEEP RESISTIVITY
+~PARAMETER INFORMATION
+#MNEM.UNIT        VALUE       DESCRIPTION
+#---------    -------------   ------------------------------
+ BHT .DEGC         35.5000:   BOTTOM HOLE TEMPERATURE
+ BS  .MM          200.0000:   BIT SIZE
+ FD  .K/M3       1000.0000:   FLUID DENSITY
+ MATR.              0.0000:   NEUTRON MATRIX(0=LIME,1=SAND,2=DOLO)
+ MDEN.           2710.0000:   LOGGING MATRIX DENSITY
+ RMF .OHMM          0.2160:   MUD FILTRATE RESISTIVITY
+ DFD .K/M3       1525.0000:   DRILL FLUID DENSITY
+~Other
+     Note: The logging tools became stuck at 625 meters causing the data
+	   between 625 meters and 615 meters to be invalid.
+~A  DEPTH     DT       RHOB     NPHI     SFLU     SFLA      ILM      ILD
+1670.000   123.450 2550.000    0.450  123.450  123.450  110.200  105.600
+1669.875   123.450 2550.000    0.450  123.450  123.450  110.200  105.600
+1669.750   123.450 2550.000    0.450  123.450  123.450  110.200  105.600

--- a/tests/test_read.py
+++ b/tests/test_read.py
@@ -181,3 +181,7 @@ def test_missing_a_section():
 def test_blank_line_in_header():
     las = read(egfn('blank_line.las'))
     assert las.curves[0].mnemonic == 'DEPT'
+
+def test_blank_line_at_start():
+    las = read(egfn('blank_line_start.las'))
+    

